.github/workflows/odoo-ci-cd.yml

```
@@ -1,41 +1,468 @@
-        uses: docker/build-push-action@v5
-        with:
-          context: .
-          push: true
-          tags: ghcr.io/${{ github.repository }}:${{ steps.vars.outputs.tag }}
+      - name: List GHCR tags and run containers
+        id: lsghcr
+        env:
+          GHCR_OWNER: my-org
+          IMAGE_NAME: my-app
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:38:13: Double quote to prevent globbing and word splitting

(shellcheck)